packages/popover/components/PopoverListItem.tsx

```diff
@@ -28,7 +28,7 @@ export interface PopoverListItemProps extends React.HTMLProps<HTMLDivElement> {
   index: number;
   listLength: number;
   isActive?: boolean;
-  isSelected?: boolean;
+  isSelected: boolean | null;
```

Comment:
Could you keep `isSelected` as an optional prop so we can retain backwards compatibility with this component?
Just in case someone may be using this component and they didn't have isSelected set, it would not cause them any errors.

If not, this will need to be in a separate `feat` commit since you are changing how the props of this component function.

Reply:
Thanks. Could you please reduce this to 1 commit that is the chore of updating the deps and the second commit will contain any code changes and that will be a `refactor:` commit. There should be only 2 commits total. If you need to you can do a `git reset HEAD~2` to fix the commits.

Reply:
After that change we can get this merged in.

Reply:
Have reduced it to single commit.